src/content/homepage.md

```diff
@@ -40,31 +40,31 @@ technologySection:
   subtitle: What makes Alephium different?
   blockFlowSection:
     title: 'Scalable & Sharded'
-    description: "Alephium is built on a novel and complete sharding algorithm called BlockFlow. It improves on the UTXO model of BTC to make it scalable and uses DAG data structure to reach consensus between different shards. This will allow up to 10,000 Transactions Per Second (currently more than 400 TPS vs Bitcoin's 7 TPS)."
+    description: "Alephium is built on a novel sharding algorithm called BlockFlow. It leverages a stateful UTXO model and DAG data structure to deliver efficient and practical sharding. This enables Alephium to natively support over 10,000 transactions per second while preserving a seamless “'single-chain” user experience."
```

Comment:
Same here, the quotes around "single-chain" will probably cause issues. It's possible to "escape" the quotes by using a backslash `\`. I can take care of it.